src/shared/components/publications-and-talks-section/videos-list/remaining-video/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { injectIntl } from 'react-intl';
+import ReactPlayer from 'react-player';
+import PropTypes from 'prop-types';
+
+import styles from './index.module.css';
+
+class RemainingVideo extends Component {
+    constructor() {
+        super();
+
+        this.handleRemainingVideoClick = this.handleRemainingVideoClick.bind(this);
+    }
+
+    render() {
+        const { video, intl: { messages } } = this.props;
+
+        return (
+            <div className={ styles.videoItemContainer } onClick={ this.handleRemainingVideoClick }>
+                <div className={ styles.videoPlaceholder }>
+                    <ReactPlayer
+                        className={ styles.reactPlayer }
+                        url={ video.link }
+                        width="100%"
+                        height="100%" />
+                </div>
+                <p className={ styles.videoDescription }>{ messages[video.description] }</p>
+            </div>
+        );
+    }
+
+    handleRemainingVideoClick() {
```

Comment:
If you declare this using an arrow function, you can delete the constructor because the bind is implicit. 

```js
handleRemainingVideoClick => () {
	// ...
}
```